wdio.conf.js

```diff
@@ -1,16 +1,14 @@
 const path = require('path');
-const defaultWdioConfig = require('terra-toolkit/config/wdio/wdio.conf');
-
-const wdioConfig = defaultWdioConfig.config;
+const { config } = require('@cerner/terra-functional-testing/lib/config/wdio.conf');
```

Comment:
I think this works now
```suggestion
const config  = require('@cerner/terra-functional-testing');
```

Reply:
I think this: 
`const { config } = require('@cerner/terra-functional-testing');`